tests/Jenkins/dirac_ci.sh

```diff
@@ -116,7 +117,8 @@ function installSite(){
 	sed -i s/VAR_DB_Port/$DB_PORT/g $SERVERINSTALLDIR/install.cfg
 
 	echo '==> Started installing'
-	$SERVERINSTALLDIR/install_site.sh install.cfg
```

Comment:
The full path in a Jenkins job is not guaranteed to be always the same, 
so there's no way but navigating. And also these tests should be able to 
run on other CIs too, like Travis. So, when dirac_ci.sh is sourced, the 
paths are created.